packages/button/README.md

```diff
@@ -342,3 +380,47 @@ popover or dialog opens.
 ```html
 <sp-button autofocus>Confirm</sp-button>
 ```
+
+### Accessibility guidelines
+
+#### Include a label
+
+A button is required to have either a visible text label or `label` attribute on an `<sp-button>`
+or on an `<sp-icon*>` element child.
+
+#### Don't override color
+
+Do not use custom colors for buttons. The colors of different button variations have been designed to be consistent and accessible.
+
```

Comment:
Proposed addition.
#### Don't mix href and non-href buttons in a set of buttons

A screen reader user will not encounter href buttons when navigating by buttons or form controls. While they can both be used in the same page problems could occur if mixing the types in close proximity to each other.